File: kumoh-code-2018/C#/#1 프로젝트/Omok/Omok(Console)/Board.cpp
#include <iostream>
#include <string>
using namespace std;

class board {
private:
	int matrix[10][10];
public:
	void putStone(int x, int y, int stone){
		matrix[x][y] = stone;
	}
	void removeStone(int x, int y){
		matrix[x][y] = 0;
	}
	int getStone(int x, int y){
		return matrix[x][y];
	}
	void display(){
		int x, y;
		cout << "¢Ù ";
		for (x = 0; x<10; x++){
			cout << x << " ";
		}
		cout << endl;
		for (x = 0; x<10; x++){
			cout << " " << x;
			for (y = 0; y<10; y++){
				if (matrix[x][y] == 0) cout << "¡à";
				else if (matrix[x][y] == 1) cout << "¡Ü";
				else if (matrix[x][y] == 2) cout << "¡Û";
				else cout << "¡à";
			}
			cout << endl;
		}
	}
	void clear(){
		int x, y;
		for (x = 0; x<10; x++){
			for (y = 0; y<10; y++){
				matrix[x][y] = 0;
			}
		}
	}
};

File: kumoh-code-2018/C#/#1 프로젝트/Omok/Omok(Network)/Omok(Network)/Omok(Game_grapic) server/Omok(Game_grapic) server/board.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Omok
{
    public class board
    {
        private const int N = 19;
        private int[,] matrix = new int[N, N];

        public void putStone(int x, int y, int stone)
        {
            matrix[x, y] = stone;
        }
        public void removeStone(int x, int y)
        {
            matrix[x, y] = 0;
        }
        public int getStone(int x, int y)
        {
            return matrix[x, y];
        }
        public void display()
        {
            int x, y;
            Console.Write("↘ ");
            for (x = 0; x < N; x++)
            {
                Console.Write(x + " ");
            }
            Console.WriteLine();
            for (x = 0; x < N; x++)
            {
                Console.Write(" " + x);
                for (y = 0; y < N; y++)
                {
                    if (matrix[x, y] == 0) Console.Write("□");
                    else if (matrix[x, y] == 1) Console.Write("●");
                    else if (matrix[x, y] == 2) Console.Write("○");
                    else Console.Write("□");
                }
                Console.WriteLine();
            }
        }
        public void clear()
        {
            int x, y;
            for (x = 0; x < N; x++)
            {
                for (y = 0; y < N; y++)
                {
                    matrix[x, y] = 0;
                }
            }
        }
    }
}